metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,84 @@
+
+### What is the dataset?
+
+### What is the purpose of the dataset?
+
+The Utah Roads layer plays an important role in active transportation planning in Utah. Recent collaborative efforts between UGRC, [UDOT](https://www.udot.utah.gov/connect/,) and [WFRC](https://wfrc.org/) have led to significant enhancements in the bike and pedestrian data on both the roads and trails/pathways layers. You can contribute to the on-street bike data by letting us know where we have gaps or inaccurate information.
+
+<!--- Is there a contact form people should use when they want to let us know there is a gap in the data? Should we direct them to the office email address? -->
+
+This data model is meant to suggest 'best practices'. It is not the intent that all stewards will change their current data management process to match this standard, but rather that stewards will evaluate their current practices to see how well their data 'translates' to this schema for data sharing and Next Generation 911 preparedness.
+
+<!--- Is it common practice for data stewards to change their data to fit ours?  -->
+
+### What does the dataset represent?
+
+<!--- How did we decide to delineate the individual polylines? I noticed that for many roads there is more than one polyline segment for one named road. -->
+
+### How was the dataset created?
+
+The Utah Roads dataset utilizes a data model based on [Next-Generation 911 standards](https://www.911.gov/issues/ng911/) and the Federal Highway Administration's All Roads Network Of Linear-referenced Data ([ARNOLD](https://highways.dot.gov/safety/data-analysis-tools/rsdp/rsdp-tools/arnold-reference-manual)) reporting requirements for state DOTs. UGRC adopted this data model on September 13th, 2017. The statewide roads dataset is maintained by UGRC in partnership with local government, UDOT, and the 911 community.
+
+### How reliable and accurate is the dataset?
+
+We are currently working with US Forest Service to improve the Forest Service roads in this dataset, however, for the most current and complete set of USFS roads, please visit the
+[FSGeodata Clearinghouse](https://data.fs.usda.gov/geodata/edw/datasets.php?dsetCategory=transportation).
+
+Please direct questions about this layer to Greg Bunce from UGRC at [gbunce@utah.gov] or by calling 801-349-0039.
+
+## Credits
+
+### Data Source
+
+- Local data stewards
+- UDOT
+- UGRC
+
+### Host
+
+UGRC
+
+## Restrictions
+
+For more information on usage limitations for this dataset, please see our [disclaimer](https://gis.utah.gov/documentation/policy/license/#disclaimer).
+
+## License
+
+CC by 4.0 License
+
+## Tags
+
+- UDOT
+
+## Secondary Category
+
+## Data Page Link
+
+[https://gis.utah.gov/products/sgid/transportation/road-centerlines/]
+
+## Update
+
+### Update Schedule
+
+Monthly
```

Comment:
**nitpick**: yes it is monthly but it should be mentioned that not every county is updated every month. If I remember correctly, there is a schedule linked to somewhere.

Reply:
I will look for that schedule, and specify that the counties updated each month vary.

Reply:
@gregbunce

Reply:
it's best to keep the term 'monthly' as a broad term. for the most part, we update about 8 counties a month (the big 5 (salt lake, utah, washington, weber, and davis) + a few rural counties based on [this schedule](https://docs.google.com/spreadsheets/d/1Gmt9oYMNjsIhyyMOWU6O9GnGZuSKxXs-zuHR7ADEBTs/edit#gid=0))

but, i don't want to put the schedule on the data page, as it's more of our internal guide/intention. out goal is follow the schedule, but tt doesn't always go as planned. Occasionally we have internal issues and other times we don't get the data from the standards. but, overall, it's pretty accurate.

Reply:
Do the benefits of knowing the schedule outweigh the unlikely fallout from customers that it wasn't met? seems as though a quick explanation would remove any grounds to complain.

Reply:
i see your point. i'll mull it over.

Reply:
Did you have a chance to give this one some thought? I am looking to find out what the general guideline will be moving forward--do we keep it vague, or do we offer specific update schedules? Take all the time you need deciding, just checking in for a status update on this topic. @gregbunce

Reply:
let's keep it off the website for now. i'm thinking we create something more useful (soon), more like a dashboard that shows what counties have contributed data, when, and the last updates. i feel like this option is more useful and also shows what counties owe us an update.

Reply:
I think a dashboard sounds like a great idea! I'll leave the update schedule off for now.